static/gsApp/views/amCheckout/cart.spec.tsx

```diff
@@ -0,0 +1,292 @@
+import moment from 'moment-timezone';
+
+import {BillingConfigFixture} from 'getsentry-test/fixtures/billingConfig';
+import {PlanDetailsLookupFixture} from 'getsentry-test/fixtures/planDetailsLookup';
+import {SubscriptionFixture} from 'getsentry-test/fixtures/subscription';
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {render, screen, within} from 'sentry-test/reactTestingLibrary';
+import {resetMockDate, setMockDate} from 'sentry-test/utils';
+
+import SubscriptionStore from 'getsentry/stores/subscriptionStore';
+import {InvoiceItemType, OnDemandBudgetMode, PlanTier} from 'getsentry/types';
+import AMCheckout from 'getsentry/views/amCheckout/';
+import Cart from 'getsentry/views/amCheckout/cart';
+import {SelectableProduct, type CheckoutFormData} from 'getsentry/views/amCheckout/types';
+
+// Jun 06 2022 - with milliseconds
+const MOCK_TODAY = 1654492173000;
+
+describe('Cart', () => {
+  const api = new MockApiClient();
```

Comment:
might need to call api.clear on this between tests. not sure what its used for

Reply:
i copied this from another test but we probably don't need it